CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# Contributing Guidelines
+
+Welcome to Create Block Theme! All are welcome here.
+
+## How can I contribute?
+
+We welcome contributions in all forms, including code, design, documentation, and triage.
+
+### Development Setup
+
+The basic setup for development is:
+
+-   Node/NPM Development Tools
+-   WordPress Development Site
+-   Code Editor
+
+#### Prerequisites
+
+-   [Node.js](https://nodejs.org/en/) (v16.9.1)
+-   We recommend using [Node Version Manager](https://github.com/nvm-sh/nvm) (nvm) to manage your Node.js versions
+-   WordPress Development Site, such as [wp-env](https://github.com/WordPress/gutenberg/blob/trunk/packages/env/README.md) or [Local](https://localwp.com/)
+
+We recommend following the [Gutenberg code contribution guide](https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/getting-started-with-code-contribution.md) for more details on setting up a development environment.
+
+#### Code Setup
+
+Fork the Create Block Theme repository, clone it to your computer and add the WordPress repository as upstream.
+
+```
+$ git clone https://github.com/YOUR_GITHUB_USERNAME/create-block-theme.git
+$ cd create-block-theme
+$ git remote add upstream https://github.com/WordPress/create-block-theme.git
+```
+
+Run the following commands to install the plugin dependencies:
+
+```
+npm install
+composer install
```

Comment:
This would require composer to be installed globally — should we add that to the pre-reqs?

Reply:
Right, I think we should. I've updated the pre-reqs here: https://github.com/WordPress/create-block-theme/pull/220/commits/dd049d330287e48fec6c0b8a7c9252fa13f81dd9